### PR TITLE
Mentorwho & msay fixes

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
add: Added mentorwho verb to see who mentors are.
fix: Fixed msay and added an F4 hotkey.
/:cl:

### CODE REVIEW

**Comment by PopNotes** [comment]
very yes

**Comment by cacogen** [comment]
when committing skin.dmf did you find you had a bunch of extra changed lines to fix?

**Comment by tortellinitony** [comment]
No, just the changes you see.

File: interface/skin.dmf [hunk 1]
```diff
@@ -486,6 +490,10 @@ macro "hotkeys"
 		command = ".screenshot"
 		is-disabled = false
 	elem 
+		name = "F4"
```
Comment (JohnGinnane) [inline interface/skin.dmf]: Can i suggest putting this after F3 rather than before?

**Review by JohnGinnane — CHANGES_REQUESTED** [review]
Code looks good, just wondering if you could put shortcuts in order

**Comment by ghost** [comment]
Code looks good, please fix what @JohnGinnane mentioned and then we can merge it 👍

**Review by JohnGinnane — APPROVED** [review]
Looks good

**Pull request merged**